examples/widgets/list/lv_example_list_2.c

```diff
@@ -0,0 +1,109 @@
+#include "../../lv_examples.h"
+#if LV_USE_LIST && LV_BUILD_EXAMPLES
+
+static lv_obj_t* list2;
+static lv_obj_t* buttonColumn;
+
+static lv_obj_t* currentButton = NULL;
+
+static void event_handler(lv_event_t* e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    lv_obj_t* obj = lv_event_get_target(e);
+    if (code == LV_EVENT_CLICKED)
+    {
+        LV_LOG_USER("Clicked: %s", lv_list_get_btn_text(list2, obj));
+
+        if (currentButton == obj)
+        {
+            currentButton = NULL;
+        }
+        else
+        {
+            currentButton = obj;
+        }
+        lv_obj_t* parent = lv_obj_get_parent(obj);
+        uint32_t i;
+        for (i = 0; i < lv_obj_get_child_cnt(parent); i++)
+        {
+            lv_obj_t* child = lv_obj_get_child(parent, i);
+            if (child == currentButton)
+            {
+                lv_obj_add_state(child, LV_STATE_CHECKED);
+            }
+            else
+            {
+                lv_obj_clear_state(child, LV_STATE_CHECKED);
+            }
+        }
+    }
+}
+
+static void event_handler_mu(lv_event_t* e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    // lv_obj_t* obj = lv_event_get_target(e);
+    if ((code == LV_EVENT_CLICKED) || (code == LV_EVENT_LONG_PRESSED_REPEAT))
+    {
+        if (currentButton == NULL) return;
+        lv_obj_t* parent = lv_obj_get_parent(currentButton);
+        uint_fast32_t i = lv_obj_get_child_id(currentButton);
+        if (i > 0)
+        {
+            lv_obj_swap(parent->spec_attr->children[i], parent->spec_attr->children[i - 1]);
+        }
+    }
+}
+
+static void event_handler_dn(lv_event_t* e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    // lv_obj_t* obj = lv_event_get_target(e);
+    if ((code == LV_EVENT_CLICKED) || (code == LV_EVENT_LONG_PRESSED_REPEAT))
+    {
+        if (currentButton == NULL) return;
+        lv_obj_t* parent = lv_obj_get_parent(currentButton);
+        uint_fast32_t i = lv_obj_get_child_id(currentButton);
+        if (i < lv_obj_get_child_cnt(parent) - 1)
+        {
+            lv_obj_swap(parent->spec_attr->children[i], parent->spec_attr->children[i + 1]);
+        }
+    }
+}
+
+
+void lv_example_list_2(void)
```

Comment:
It should be declared in [lv_example_widgets.h](https://github.com/lvgl/lvgl/blob/master/examples/widgets/lv_example_widgets.h).

Reply:
ok